### ISSUE TITLE
!status should not confirm the existence of certain roles

### ISSUE THREAD

**Issue by codejr** [chrisgillis/slackwolf#55]
Status command confirms if there is a bodyguard in the game. Hard to lie about it.

**Comment by chrisgillis** [comment]
Good point. Could just simply to show number of remaining actions needed.

```
!status
Night [ 2 more actions needed]
```

This still gives away number of werewolves at a minimum.

So we could consider adding `!option status on|off` as well for groups who find that undesirable.

**Comment by chrisgillis** [comment]
Ran into this in a game last night :P

**Comment by chrisgillis** [comment]
We also found the need for a weighted role strategy... Smaller games of 5 players or so tend to skew heavily in favor of the villagers points wise.
